src/windows/mod.rs

```diff
@@ -530,10 +530,18 @@ impl Window<IambInfo> for IambWindow {
                 Spans::from(title)
             },
             IambWindow::MemberList(_, room_id, _) => {
-                let title = store.application.get_room_title(room_id.as_ref());
-
-                Spans(vec![bold_span("Room Members: "), title.into()])
-            },
+                let v = match store.application.worker.members(room_id.to_owned()) {
```

Comment:
Fetching the list of members is unfortunately expensive in big rooms. We don't want to do that every time we print the title, so this will need to get the length of the list from the `ListState`. I'm going to add a new method to take care of that over in ulyssa/modalkit#116, which you can then use here.

You'll also want to make sure to update the `get_win_title()` method.

Reply:
I just pushed a commit for ulyssa/modalkit#116. If you want, you can point the `Cargo.toml` at the Git ref on Github and I'll update it once I cut a modalkit release.